articles/notification-hubs/android-sdk.md

```diff
@@ -196,18 +196,17 @@ also have the connection strings that are necessary to send notifications to a d
    public class CustomNotificationListener implements NotificationHubListener {
 
       @override
-
-      public void onNotificationReceived(Context context, NotificationMessage message) {
-
-      /* The following notification properties are available. */
-
-      String title = message.getTitle();
-      String message = message.getMessage();
-      Map<String, String> data = message.getData();
-
-      if (message != null) {
-         Log.d(TAG, "Message Notification Title: " + title);
-         Log.d(TAG, "Message Notification Body: " + message);
+      public void onNotificationReceived(Context context, RemoteMessage message) {
+    
+         /* The following notification properties are available. */
+         Notification notification = message.getNotification();
+         String title = notification.getTitle();
+         String body = notification.getBody();
+         Map<String, String> data = message.getData();
```

Comment:
nit that you didn't even introduce - `data` is unused at the moment. Mind updating the logging below to print the number of keys in the map or something?